reports/summary-2021.md

```diff
@@ -0,0 +1,1195 @@
+part of the annual report process should ensure that project contributing docs
+remain (relatively) up-to-date.
+
+
+[auditing]: https://github.com/kubernetes/community/issues/5229
+
+
+#### Areas to research 
+
+Updating documentation is usually a good onboarding path for interns and 
+new contributors but this can get murky with some of the complexities of
+the code and doc set. It can take up to 3 months to onboard on to the project
+before suggestion and submitting changes. Is there a program that SIGs could
+create as an onboarding path towards OWNERship here? 
+
+
+
+#### Burnout 
+
+The topics of burnout and workload management are frequent in our Leads and 
+group meetings, Steering Committee, and even the growing voices at ecosystem
+level during talks and events at KubeCon/CloudNativeCons. This is an industry
+wide problem that we need to solve together. With a mix of reasons why 
+contributors are burning out, there is no one solve all solution here. 
+Aligning incentives to grow OWNERs seems to be one of the main challenges in 
+this space. 
+
+#### What have we done
+- Reducing the release cadence. While this wasn't the only reason for having 
+3 and not 4 releases in a year, it factored in. [Check out this blog for more on the release cadence change](https://kubernetes.io/blog/2021/07/20/new-kubernetes-release-cadence/).
+- Continue to talk about how we can do better in our monthly Chairs and Tech Leads and keeping our doors open for contributors to have these conversations with us.
+- Keep educating contributors and OWNERs on the use of "Emeritus" and why it's ok to take breaks.
+
+
+
+
+### Growth Areas
+
+This section represents an area of the project that we've identified as having a growth opportunity or need.
+
+
+
+#### What's project health anyway?
+
+Some of the more mature groups like SIG Instrumentation or those with industry
+open-source veterans can quickly identify areas of their components that need
+help and tell stories about what's flourishing. Yet, it can be challenging to
+establish universal indicators of "project health" in a project as large and
+diverse as Kubernetes. We need to develop these indicators to provide signal to
+the leads so that they may detect, pre-empt, or bubble up this information to
+keep their area healthy.
+
+
+#### Every group needs more reviewers
+
+If you've been watching open source news over the last year, supply chain security has made headlines. According to OpenSSF and other security groups, [insert line about reviewers are important] 
+Reviewers are a key part of our success in quality code and documentation changes upstream. Reviewer is the next step on our [contributor ladder](https://github.com/kubernetes/community/blob/master/community-membership.md) post member 
+- talk about showing up and sticking around
+- talk about chopping wood and carrying water first 
+- maybe include the reviewer graph stats 
+
```

Comment:
As a data point, in 1.19, the average daily PR reviewers were down -15% across Kubernetes Orgs and -24% in the Kubernetes/Kubernetes repo while still receiving the same amount of issues and PRs. If you are trying to get a change into Kubernetes, this will effect you.